### PR TITLE
I#99: API for finance/history

### PR DESCRIPTION
**Pull request by shreyansjain**
#99

### CODE REVIEW

**Comment by shreyansjain** [comment]
Wiki here: https://github.com/PayUppApp/PayUpp-AWS/wiki/Team-Finance#get-teamuserhistory

**Comment by shreyansjain** [comment]
Just doing (amount * quantity) and sending as amount since the mockups do not refer to quantity per say for account history. however, its good we are capturing the quantity info

File: api-team/functions/finance/history/get/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    Promise.coroutine(function* () {
+
+        const team_id = event.pathParameters.team_id;
+        let UserObj = new User().convertToClassObj(claimsObj);
+
+        console.log(UserObj);
+        let client = yield pool.connect();
+        try {
+            let user_finance_history = yield client.query(SQL `
+                    SELECT cc.description, (tf.amount * tf.quantity) as amount, tf.TRANSACTION_DATE
```
Comment (anmolg) [inline api-team/functions/finance/history/get/index.js]: I thought we were storing total amount anyways? So we don't have to multiply?

File: api-team/serverless.yml [hunk 2]
```diff
@@ -147,4 +147,17 @@ functions:
                 authorizer:
                     arn: arn:aws:cognito-idp:us-west-2:949036265161:userpool/us-west-2_tCFjmV5Zs
 
+    # sls invoke local --function GetUserTeamFinanceHistory --path functions/finance/history/get/event.json -s local
+    # Deposit amount for a user for a team
+    GetUserTeamFinanceHistory:
+        handler: functions/finance/history/get/index.handler
+        events:
+            - http:
+                path: team/{team_id}/finance/history
```
Comment (anmolg) [inline api-team/serverless.yml]: There must also be a historys ection for user

**Comment by shreyansjain** [comment]
Updated the wiki

**Pull request merged**